Documentation/gettingstarted/k8s-install-aks-create-cluster.rst

```diff
@@ -0,0 +1,19 @@
+.. note:: **Do NOT specify the '--network-policy' flag** when creating the cluster,
+    as this will cause the Azure CNI plugin to push down unwanted iptables rules:
+
+.. code:: bash
+
+   export RESOURCE_GROUP_NAME=aks-test
+   export CLUSTER_NAME=aks-test
+   export LOCATION=westeurope
+
+   az group create --name $RESOURCE_GROUP_NAME --location $LOCATION
+   az aks create \
+      --resource-group $RESOURCE_GROUP_NAME \
+      --name $CLUSTER_NAME \
+      --location $LOCATION \
+      --node-count 2 \
+      --network-plugin azure
```

Comment:
What's the rationale for removing `--generate-ssh-keys`? I have been bitten many times in the past that nodes became inaccessible, since I did not have any way to access the host. 

While it is not strictly necessary, I strongly recommend it to first-time users, so in case of issues we can troubleshoot the nodes. Users who know what they are doing will anyway adapt the command if necessary.

Reply:
The default in AKS is no SSH, and personally I believe that running managed Kubernetes with SSH enabled is not a good idea. Additionally, AKS nodes are on a private network, so extra steps will needed to enable access.

Is it standard practice for Cilium troubleshooting to require user to SSH? My impression was that sysdump tool, and `kubectl exec` to Cilium pod where the main methods.

Reply:
Having administrated a bunch of AKS cluster myself in the past, I can tell you that SSH access is crucial for troubleshooting. You may call it an anti-pattern, but there is just no alternative to running AKS in prod. But I don't have a strong opinion, I'm fine to keep the defaults hat Azure promotes.

Reply:
Yes, let's stay with AKS default for most part. Having `--node-count` lowered from 3 to 2 makes sense, as it's cheaper. I've not used AKS myself until yesterday, so cannot say as much to debate practical AKS ops.